cmd/carbonapi/http/find_handlers.go

```diff
@@ -192,7 +192,7 @@ func findHandler(w http.ResponseWriter, r *http.Request) {
 	query := r.Form["query"]
 	srcIP, srcPort := splitRemoteAddr(r.RemoteAddr)
 
-	accessLogger := zapwriter.Logger("access")
+	accessLogger := zapwriter.Logger("find")
```

Comment:
Not sure if that's required.

Purpose of accessLogger is to produce exactly one message per request (as you'd expect from access logs in http servers) with all possible request stats. And it's named "access" on purpose as usually you either want all access logs or none of them. However I'm open to discuss that if you think that it would be worth to silence only access logs for specific handlers. But even in that case logs are sort of cheap (in terms of cost to call .Info in zap) so I'd suggest to add a warning on startup that behaviour will be changed soon (and only if some one have access logger redefined in config), update the doc, create a separate logger per handler and log to both access and that separate logger.